translator/GCP_cloud_function/cloud_function/main.py

```diff
@@ -52,18 +52,17 @@ def parse_xml(xml_string):
 def get_wzdx_schema(schema_file_name):
   return json.loads(open(schema_file_name,'r').read())
 
-#this function retrieves username and password info from secrets .This will be implemented after correct secrets are implemented.
 def get_ftp_credentials():
   secret_client = secretmanager.SecretManagerServiceClient()
-  username_secret_name = os.environ['icone_ftp_username_secret_name']
-  password_secret_name = os.environ['icone_ftp_password_secret_name']
+  username_secret_name = os.environ['icone_ftp_username_secret_name']#"icone_ftp_username"
```

Comment:
Can we remove these comments? They don't seem to be explaining or helping.

Reply:
I removed those comments and pushed the updated code.